DOCS.md

```diff
@@ -130,3 +130,38 @@ Cannot connect to the Docker daemon. Is 'docker -d' running on this host?
 ```
 
 The above issue can be resolved by setting `storage_driver: vfs` in the `.drone.yml` file. This may work, but will have very poor performance as discussed [here](https://github.com/rancher/docker-from-scratch/issues/20).
+
+
+This error occurs when using Debian wheezy or jessie which kernel misses some features:
+
```

Comment:
I agree with @bradrydzewski that specifying any known immediate limitations that might be addressed later on would be great, but otherwise LGTM.